addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -31,530 +20,95 @@ class ComposerTextInput extends Component {
          * Max frequency for searching for mention suggestions. Useful to
```

Comment:
The comment should be removed with the line that is removed.

Reply:
Done